torch/csrc/utils/tensor_new.cpp

```diff
@@ -130,16 +130,52 @@ static Tensor new_from_data(const Type & type, int device, PyObject *data) {
   }
 }
 
+static Tensor sparse_tensor_new(const Type& type, PyObject* args, PyObject* kwargs) {
+  static PythonArgParser parser({
+    "new(*, int64_t? device=-1)",
+    "new(IntList size, *, int64_t? device=-1)",
+    "new(*, int64_t cdata)|hidden",
+    "new(Tensor indices, Tensor values, *, int64_t? device=-1)",
```

Comment:
should we have a single Tensor overload that only works if you pass in a sparse tensor?

Reply:
We don't support that currently. I'd rather not add additional overloads since I think we're planning on moving away from these constructors.